lib/vintage_net_wizard/watch_dog.ex

```diff
@@ -0,0 +1,49 @@
+defmodule VintageNetWizard.WatchDog do
+  @moduledoc false
+
+  # This server is used to watch for activity in the wizard. If the inactivity
+  # timeout is triggered this server will shutdown the wizard. In order to keep
+  # this server from shutting down the wizard call `pet/0` to reset the
+  # timeout. The timeout should be in minutes.
+
+  use GenServer
+
+  require Logger
+
+  @doc """
+  Start the watcher
+  """
+  @spec start_link(timeout()) :: GenServer.on_start()
+  def start_link(inactivity_timeout_seconds) do
+    GenServer.start_link(__MODULE__, inactivity_timeout_seconds, name: __MODULE__)
+  end
+
+  @doc """
+  Pet the watch dog to prevent it from shutting down the wizard
+  """
+  @spec pet() :: :ok
+  def pet() do
+    GenServer.call(__MODULE__, :ping)
```

Comment:
```suggestion
    GenServer.call(__MODULE__, :pet)
```

This is a nit...but if the function is `pet`, I feel like the GenServer call should be as well. Thoughts?

Reply:
Good catch! Updated!